File: D758.java
import java.util.Scanner;

/**
 * D759
 */
public class D758 {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int casos = sc.nextInt();
        while (casos-- != 0) {
            int n = sc.nextInt();
            int negras = ((n - 1) * 2 - 1) + n * 4 - 6;
            int blancas = (int) Math.pow(n, 2) - negras;
            System.out.println(negras + " " + blancas);
        }
    }
}

File: D758.c
#include <stdio.h>

int main() {
    int casos;
    scanf("%d", &casos);
    while (casos--) {
        int n;
        scanf("%d", &n);
        int negras = ((n - 1) * 2 - 1) + n * 4 - 6;
        int blancas = n * n - negras;
        printf("%d %d\n", negras, blancas);
    }
}